narwhals/_dask/expr.py

```diff
@@ -641,9 +674,65 @@ def func(df: DaskLazyFrame) -> list[Any]:
             root_names=self._root_names,
             output_names=self._output_names,
             returns_scalar=False,
+            modifies_index=False,
             backend_version=self._backend_version,
         )
 
+    def sort(self: Self, *, descending: bool = False, nulls_last: bool = False) -> Self:
+        na_position = "last" if nulls_last else "first"
+
+        def func(_input: Any, ascending: bool, na_position: bool) -> Any:  # noqa: FBT001
+            import dask_expr as de  # ignore-banned-import
+
+            name = _input.name
+
+            result = _input.to_frame(name=name).sort_values(
+                by=name, ascending=ascending, na_position=na_position
+            )[name]
+            return de._collection.Series(de._expr.AssignIndex(result, _input.index))
+
+        return self._from_call(
+            func,
+            "sort",
+            not descending,
+            na_position,
+            returns_scalar=False,
+            modifies_index=False,
+        )
+
+    # Index modifiers
+
+    def drop_nulls(self: Self) -> Self:
+        return self._from_call(
+            lambda _input: _input.dropna(),
+            "drop_nulls",
+            returns_scalar=False,
+            modifies_index=True,
+        )
+
+    def head(self: Self, n: int) -> Self:
+        return self._from_call(
+            lambda _input, _n: _input.head(_n, compute=False),
+            "head",
+            n,
+            returns_scalar=False,
+            modifies_index=True,
+        )
+
+    def tail(self: Self, n: int) -> Self:
+        return self._from_call(
+            lambda _input, _n: _input.tail(_n, compute=False),
```

Comment:
So, `head` has a `npartitions` param which can be set to -1 and scan all partitions, while `tail` does not.

Reply:
I haven't seen cases where users actually want more than 1 partition if they call head or tail tbh, Yes this is technically an issue, but not something I've encountered in the wild